### ISSUE TITLE
[Feature]Inviter igen efter tidligere invitation er udløbet

### ISSUE THREAD

**Issue by mhewel** [CodingPirates/forenings_medlemmer#1106]
## Overordnet beskrivelse af funktionen
Forespørgsel fra en lokalforening om man kan sende invitation igen (efter tidligere er udløbet).
Systemet blokerer for det idag.

**Comment by lakridserne** [comment]
Bør ikke implementeres som beskrevet. Det er bevidst at vi blokerer det. Den gode måde at gøre det på er at forlænge invitationen. Det kan vi evt. lave en action til.

**Comment by mhewel** [comment]
Mon vi kan gøre en mellemting, dvs hvis man vil invitere, og en person allerede har fået en invitation, så bliver denne inviation forlænget til den nye dato , og selvfølgelig informere om det i resultatlisten ?

**Comment by lakridserne** [comment]
Nej. Det vil være et problem hvis man har udvalgt en række personer, hvor man ønsker at invitere dem, som ikke allerede er inviteret. Og nej, det kan ikke løses med et filter, fordi vi i så fald ville ændre noget eksisterende funktionalitet, hvor nogle folk er blevet vænnet til at arbejde på en bestemt måde.

Som beskrevet er jeg åben overfor en separat action, som forlænger invitationer.
